app/views/errors/internal_server_error.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= content_for :title, page_title('internal_server_error') %>
```

Comment:
It's good practice to create 500 pages as plain HTML with inline styles, as this reduces the risk of an infinite loop if the error page triggers an error which triggers an error page etc

Reply:
Discussed offline, in a nutshell while true that this relies on Rails working to some extent in order to render these pages, the extent that we rely on it are fairly stable and will be unlikely to reach the production environment in a broken state due to smoke testing. So it's fine for now, with the idea that we can revisit this in the future when we harden the service past Private Beta.